platform/linux-generic/odp_ipsec_sad.c

```diff
@@ -745,8 +745,18 @@ odp_ipsec_sa_t odp_ipsec_sa_create(const odp_ipsec_sa_param_t *param)
 	}
 
 	if (odp_crypto_session_create(&crypto_param, &ipsec_sa->session,
-				      &ses_create_rc))
+				      &ses_create_rc)) {
```

Comment:
This changes linux-gen implementation too even though the commit message claims this is a validation test change. It would anyway be better to separate validation test changes from implementation changes, when possible.